message_center_compassion/models/gmc_action.py

```diff
@@ -32,6 +32,7 @@ class GmcAction(models.Model):
     """
     _name = 'gmc.action'
 
+    action = fields.Char()
```

Comment:
Why not use the field `name` already present ? I don't feel this field is needed.